doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+UTxO set in order to keep the amount of indexed information proportional to the 
+UTxO set. The component is not meant to index historical information, contrary to 
+the purpose of cardano-db-sync. 
+
++--------------+---------------------------------------------------------------+
+| Plutus chain index                                                           |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-chain-index-core``                       |
+|              | ``plutus-apps/plutus-chain-index``                            |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-script-utils                                           |
++--------------+---------------------------------------------------------------+
+| State        | In the process of being deprecated in favor of Marconi.       | 
++--------------+---------------------------------------------------------------+
+
+7. Contract monad (aka contract API, plutus contract)
+-----------------------------------------------------------
+
+`Contract monad <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library for writing Plutus contracts. It provides the Contract Monad interface 
+for writing the off-chain part of a Plutus application that is to be interpreted 
+by an emulator or a plutus application backend (PAB). 
+
++--------------+---------------------------------------------------------------+
+| Contract monad                                                               |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-ledger-constraints, plutus-chain-index-core            |
++--------------+---------------------------------------------------------------+
+| State        | Maintained but not extended. Unstable interface.              | 
++--------------+---------------------------------------------------------------+
+
+8. Contract monad emulator
+-----------------------------------------------------------
+
+`Contract monad emulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library that provides an environment for emulating the blockchain. The environment 
+provides a way for writing traces for the contract which are sequences of actions 
+by simulated wallets that use the contract. The component is highly dependent on the 
+Contract Monad. However, there are plans to decouple it from the Contract Monad 
+so that it can be used with other libraries.
+
```

Comment:
I would eliminate the last sentence until we really know what the plan is.

Reply:
I've removed that last sentence.